<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSScriptEditor</class>
 <widget class="QMainWindow" name="GSScriptEditor">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeekSpace Редатор сценариев</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/script</normalon>
   </iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QMdiArea" name="mdiArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="viewMode">
        <enum>QMdiArea::TabbedView</enum>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="tabsMovable">
        <bool>true</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpenScript"/>
    <addaction name="actionOpenFromFile"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScript"/>
    <addaction name="actionSaveScriptAs"/>
    <addaction name="separator"/>
    <addaction name="actionSaveInFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAboutLanguage"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionCheck"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Окно</string>
    </property>
    <addaction name="actionConsole"/>
    <addaction name="actionActions"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockActions">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Действия и переменные:</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="treeActions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>491</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Объекты</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpenScript"/>
   <addaction name="separator"/>
   <addaction name="actionSaveScript"/>
   <addaction name="actionSaveScriptAs"/>
   <addaction name="actionSaveDraft"/>
   <addaction name="separator"/>
   <addaction name="actionCheck"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QDockWidget" name="dockConsole">
   <property name="windowTitle">
    <string>Консоль</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTextEdit" name="console">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>871</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionSaveScript">
   <property name="icon">
    <iconset>
     <normalon>:/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Сохранить сценарий</string>
   </property>
   <property name="toolTip">
    <string>Сохранить сценарий</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveScriptAs">
   <property name="icon">
    <iconset>
     <normalon>:/saveAs</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Сохранить сценарий как</string>
   </property>
   <property name="toolTip">
    <string>Сохранить сценарий как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/newScript</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveInFile">
   <property name="text">
    <string>Сохранить в файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionOpenScript">
   <property name="icon">
    <iconset>
     <normalon>:/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Открыть сценарий</string>
   </property>
   <property name="toolTip">
    <string>Открыть сценарий</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenFromFile">
   <property name="text">
    <string>Открыть из файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normalon>:/undo</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normalon>:/redo</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Найти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAboutLanguage">
   <property name="text">
    <string>Об языке</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="icon">
    <iconset>
     <normalon>:/icons/checkScript.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normalon>:/icons/runScript.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Консоль</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionActions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Действия и переменные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionSaveDraft">
   <property name="icon">
    <iconset>
     <normalon>:/saveDraft</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Сохранить как черновик</string>
   </property>
   <property name="toolTip">
    <string>Сохранить как черновик</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
